common/app/views/fragments/contentFooter.scala.html

```diff
@@ -14,7 +14,10 @@
     @ContentFooterContainersLayout(content.content, isPaidContent) {
         @fragments.storyPackagePlaceholder(content, related, isPaidContent)
     } {
-        @fragments.onwardPlaceholder(isPaidContent)
+        @if(!(Commercial.isAdFree(request) && isPaidContent)) {
```

Comment:
Do you need the `isPaidContent` condition as well?  It wasn't there before.

Reply:
Actually that's a fair question. Let me investigate a bit.

Reply:
No, I think this makes sense. We want to show paid content to non-ad-free users only, and we want to show non-paid content regardless of ad-free. So,

```
ad-free    isPaid    result
F              F            T
F              T            T
T              F            T
T              T            F
```

Reply:
Yes I see.  I misread the brackets and didn't see that the the not covers both!